internal/provider/schema_gitlab_user_keys.go

```diff
@@ -0,0 +1,37 @@
+package provider
+
+import (
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func gitlabUserSSHKeySchema() map[string]*schema.Schema {
```

Comment:
💡 **(non-blocking) Nitpick:** Can we make this re-usable with what we have in the resource?

https://github.com/gitlabhq/terraform-provider-gitlab/blob/6540e190e384a86c4fc61da32f86332db1b9d175/internal/provider/resource_gitlab_user_sshkey.go#L30

Reply:
Done